Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/Fields.cs

```diff
@@ -23,10 +24,49 @@ namespace Relativity.Test.Helpers.ArtifactHelpers
 	/// 
 	public class Fields : IFieldsHelper
 	{
+		private static bool? _keplerCompatible;
 
-		#region DbContext Methods
+		#region Public Methods
 
 		public static int GetFieldArtifactID(String fieldname, IDBContext workspaceDbContext)
+		{
+			var keplerHelper = new KeplerHelper();
+
+			if (keplerHelper.ForceDbContext()) return GetFieldArtifactIDWithDbContext(fieldname, workspaceDbContext);
```

Comment:
Our usual standard (coming from Chandra's usual notes) is that we return at the end and not in the middle of a function.